openfe/setup/alchemical_network_planner/relative_alchemical_network_planner.py

```diff
@@ -315,6 +317,47 @@ def __init__(
             protocol=protocol,
         )
 
+    def _build_transformation(
+        self,
+        ligand_mapping_edge: LigandAtomMapping,
+        stateA: ChemicalSystem,
+        stateB: ChemicalSystem,
+        transformation_protocol: Protocol,
+    ) -> Transformation:
+        """
+        Overwrite the default method to handle net charge change transformations with our default protocol.
+        """
+        transformation_name = self.name + "_" + stateA.name + "_" + stateB.name
+
+        protocol_settings = transformation_protocol.settings.unfrozen_copy()
+        if "vacuum" in transformation_name:
+            protocol_settings.forcefield_settings.nonbonded_method = "nocutoff"
+        elif get_alchemical_charge_difference(ligand_mapping_edge) != 0:
```

Comment:
Maybe close to zero would be better? E.g. 1e-3?

Reply:
I think this should be fine as the `rdkit.Chem.rdmolops.GetFormalCharge` returns an int the difference between them should also be an int, it also seems to work in the test, I might have missed something though.

Reply:
Thanks for clarifying, sounds good!